src/OpenyActivityFinderSolrBackend.php

```diff
@@ -1053,7 +1053,6 @@ private function doSessionsSearchRequest($session_ids) {
     $query->setParseMode($parse_mode);
     $query->addCondition('status', 1);
     $query->addCondition('nid', $session_ids, 'IN');
-    $query->accessCheck(FALSE);
```

Comment:
fix for 
![image](https://github.com/YCloudYUSA/yusaopeny_activity_finder/assets/744406/f032d699-4348-4567-a80c-17c72c10f147)

Reply:
Then can we use something like:

```
    // Manually bypass access checks
    $query->setOption('search_api_bypass_access', TRUE);
```
    Instead of just removing accessCheck ?

Reply:
@rollki, thank you. You're right. FIxed